### ISSUE TITLE
Does not detect a violation when calling function is decorated

### ISSUE THREAD

**Issue by bmacl** [mobility-university/flake8-newspaper-style#8]
```python
import functools


def decorator(func):
    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        return func(*args, **kwargs)

    return wrapper


def text():
    ...

@decorator
def headline():
    return text()
```

Expect a `NEW100` error since text is defined before it is called but this passes.

**Comment by lindt** [comment]
thx for finding.
v0.4.3 should solve this.
added a test case for it.

```python
>>> lint('''
... def deco():
...     ...
... def text():
...     ...
... @deco
... def headline():
...     text()
... ''')
8:4 NEW100 newspaper style: function text defined in line 4 should be moved down

```

please try it out.

**Comment by lindt** [comment]
close the issue for now.

please reopen, if this issue still occurs for you. Thanks.
